_extensions/qrcode/qrcode.lua

```diff
@@ -19,6 +19,12 @@ local function addDependencies()
   }
 end
 
+---Add qrcode js dependencies.
+local function addDependenciesTex()
+  quarto.doc.include_text("in-header", "\\usepackage[]{qrcode}")
+end
```

Comment:
Just a head up we have specific API for adding dependency in Quarto 

https://quarto.org/docs/extensions/lua-api.html#dependencies

Here: `quarto.doc.use_latex_package(pkg, opt)`

Note: we use snake_case for our Lua API now. But old function name still works `quarto.doc.addHtmlDependency`

Reply:
thanks for the heads up, I switched to the new api now.